### PR TITLE
Marc cam/polish models

### PR DESCRIPTION
**Pull request by marclave**

### CODE REVIEW

**Comment by hanspagel** [comment]
Awesome! Why aren’t you using the `SectionColumn` component? 🤔

**Comment by marclave** [comment]
> Awesome! Why aren’t you using the `SectionColumn` component? 🤔

vscode liveshare only had saved our css removed, we refactored the html. good catch! 

we wanted to do a full width version instead

File: packages/api-reference/src/components/Content/Introduction/Introduction.vue [hunk 1]
```diff
@@ -25,12 +25,12 @@ const { state, getClientTitle, getTargetTitle } = useTemplateStore()
...
-        :loading="!info.title"
-        tight>
-        {{ info.title }}
-      </SectionHeader>
       <SectionContent :loading="!info.description && !info.title">
+        <SectionHeader
+          :loading="!info.title"
+          tight>
+          {{ info.title }}
+        </SectionHeader>
```
Comment (hanspagel) [inline packages/api-reference/src/components/Content/Introduction/Introduction.vue:33]: This breaks the loading skeleton.

Comment (marclave) [reply packages/api-reference/src/components/Content/Introduction/Introduction.vue]: cam fixed it with css

The motivation is the content is confined to the column and doesnt overflow into the second column

**Pull request merged**